src/server/api/routes/users.router.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const usersController = require('../controllers/users.controller');
+
+const {
+  checkIfAuthenticated,
+} = require('../lib/middleware/authentication.middleware');
+
+// ENDPOINT: /api/users/role :GET to get user's role for specific uid/user
+router.get('/role', checkIfAuthenticated, (req, res, next) => {
+  usersController
```

Comment:
Since we are using linter and Prettier, maybe these could be set to allow only one of the two async syntax approaches.